libraries-http-2/src/main/java/com/baeldung/okhttp/download/BinaryFileDownloadApplication.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.okhttp.download;
+
+import okhttp3.OkHttpClient;
+
+import java.io.FileOutputStream;
+
+public class BinaryFileDownloadApplication {
```

Comment:
I think we can remove this class? It isn't mentioned in the article and its better to use unit tests to drive and test our functionality.

Reply:
Definitely, I will remove it, thank you.